README.md

```diff
@@ -12,10 +12,11 @@ Get your dev environment set up (PC or Mac):
 * [Install Node](https://nodejs.org/en/download/)
 * Choose your favorite IDE, e.g. [Visual Studio Code](https://code.visualstudio.com/).
 
-Install the Ionic Framework, Cordova and Typings using npm. Note that we are using ionic 2, which is currently in beta.
+Install the Ionic Framework and Cordova using npm. Note that we are using ionic 2, which is currently in beta.
 
 ```bash
-$ npm install -g ionic@2.0.0-beta.10 cordova@6.2.0 gulp@3.9.1 tslint@3.13.0 typescript@1.8.10

```

Comment:
Yes, I think those should get installed as dependencies of the packages in package.json. I was just copying what the latest ionic2 templates have in their build process and they only talk about installing cordova and ionic globally.